README.md

```diff
@@ -1,18 +1,27 @@
 Pulp 3 RPM plugin prerequisites
 ===============================
 
-This role installs prerequisites for pulp-rpm plugin use.
+This role installs prerequisites for pulp-rpm plugin use, when installed by
+ansible-pulp .
```

Comment:
I thinks there is a space before a period, is it on purpose?

Reply:
I normally add a space if the text will be copied & pasted. That is not true here though. I'll remove it.